public/pages/LogTypes/components/LogTypeForm.tsx

```diff
@@ -40,17 +42,21 @@ export const LogTypeForm: React.FC<LogTypeFormProps> = ({
   onConfirm,
 }) => {
   const [nameError, setNameError] = useState('');
+  const [categoryError, setCategoryError] = useState('');
+  const [categoryTouched, setCategoryTouched] = useState(false);
 
-  const updateErrors = (details = logTypeDetails) => {
+  const updateErrors = (details: LogTypeItem, onSubmit = false) => {
     const nameInvalid = !validateName(details.name, LOG_TYPE_NAME_REGEX, false /* shouldTrim */);
```

Comment:
Not blocking: Is `/* shouldTrim */` dev code that should be removed at some point? Looks like it could just be an argument for the `validateName` function called `shouldTrim`, so I just wanted to clarify.

Reply:
No, it was added so that it is easy to know what that boolean stands for